---
layout: page
excerpt: "Experience"
---

**Phase Genomics, Seattle, WA**
_Senior Data Scientist, January 2024 - present_
_Data Scientist III, January 2023 – December 2023_
_Data Scientist II, July 2022-December 2022_
_Data Scientist, July 2021-June 2022_
_Bioinformatics Research Analyst, December 2020-June 2021_
-	Develop shallow and deep learning models to predict structural variation from proximity ligation sequencing data, such as convolutional neural networks to detect mutational breakpoints from images of contact frequency heatmaps and support vector machines to predict chromosomal ploidy from coverage data
-	Spearheaded efforts to improve single nucleotide variant (SNV) and copy neutral loss of heterozygosity prediction via retraining DeepVariant on proximity ligation sequencing data and developing SNV and minor allele frequency data simulation techniques
-	Deploy tools for variant prediction and interpretation, such as developing Amazon Web Services (AWS) Lambdas via Terraform for generating visual and text reporting of variant predictions and streamlining Snakemake workflows for data generation and simulation run on AWS EC2 instances
-	Prior to my promotion to Data Scientist, I assembled, scaffolded, and phased genomes for a variety of plant, animal, and fungal species, coordinating with customers to provide guidance on experimental design for unique genomes and aid in interpretation of results

**Portland VA Research Foundation, Portland, OR**
_Research Associate, January 2018 – December 2020_
I was the primary developer and data analyst for the team of an early career Principal Investigator, giving me the opportunity to take the lead on study design, independently learn analysis techniques for new projects, and coordinate with graduate students to help support their work in addition to my own.
-	Developed and benchmarked open-source, extensively unit-tested, pip installable python software for neoepitope prediction, and mentored college and high school interns in development projects for incorporation into the software package.
-	Trained random forest and support vector machine models using sklearn to aid in the prediction of immunotherapy outcomes by characterizing proteasomal cleavage sites based on protein sequence context
-	Analyzed publicly available sequencing data from a meta-cohort of cancer patients to identify genomic predictors of response to immune checkpoint inhibitor treatment, demonstrating that commonly relied upon bulk metrics such as tumor mutational burden are only weak predictors of immunotherapy response
-	Fluently used state of the art genomics software, personalized python scripts, R-based statistical analysis, bash scripting, and job scheduling for parallel computing

**Oregon Health and Science University, Portland, OR**
_Computational Biology Intern, January 2017 – December 2017_
-	Developed a neoepitope prediction method that improved upon existing methods by incorporating phasing of variants in shared haplotypes
-	Identified population-level distributions of neoepitopes in genomic data from The Cancer Genome Atlas and evaluated criteria for improved prioritization of predicted neoepitopes
-	Used and adapted of a variety of bioinformatics software for sequence alignment, variant calling, HLA type prediction, neoepitope prediction, and more

**University of Oregon, Eugene, OR**
_Graduate Teaching Fellow, Department of Biology, January 2015 – December 2016_
•	Led two weekly laboratory sections of 16-32 students each for undergraduate biology courses
•	Met with students in office hours and by appointment to provide guidance on assignments and stimulate understanding of course material
•	Evaluated and provided constructive feedback on student papers, presentations, tests, and lab reports	
_Undergraduate Research Assistant, January 2012 – December 2014_
•	Consulted with Principal Investigators and graduate students to maintain essential operations of the laboratory
•	Mentored new employees on laboratory regulations and organized sessions to work on collaborative projects
•	Presented at several laboratory meetings each year on scientific topics of interest, such as vector-borne diseases

